libxrdp/xrdp_orders_rail.h

```diff
@@ -19,6 +19,8 @@
 #if !defined(_XRDP_ORDERS_RAIL_H)
 #define _XRDP_ORDERS_RAIL_H
 
+#include "libxrdp.h"
+
```

Comment:
Depending on your view of the above comment you may or may not wish to replace this include with:-

```c
struct xrdp_orders;
struct rail_icon_info;
struct rail_monitored_desktop_order;
struct rail_notify_state_order;
struct rail_window_state_order;
```

```

Reply:
Well, that's getting to be a bit much. Above, the list directive is simple and pretty atomic. I believe that any reasonable developer would know what's going on with that and it's unlikely to change much if ever.

However, not with this. This is more complex and more prone to change, so I'd prefer to keep this one...

Reply:
Fine with me.